app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -644,55 +644,6 @@ static void updateCalculatedValueToWithinMinMax(BgReading bgReading) {
         Log.i(TAG, "NEW VALUE CALCULATED AT: " + bgReading.calculated_value);
     }
 
-    // Used by xDripViewer
```

Comment:
This is also a code cleanup not related to the aim of this PR. Please move it to different PR. The same code structure exists in three more files.